src/status_im/ui/screens/add_new/models.cljs

```diff
@@ -12,11 +12,8 @@
     (universal-links/handle-view-profile cofx data)
     (or (universal-links/handle-url cofx data)
         {:utils/show-popup {:title      (i18n/label :t/unable-to-read-this-code)
-                            :content    (i18n/label :t/use-valid-qr-code {:data data})
-                            :on-dismiss #(re-frame/dispatch [:navigate-to-clean :home])}})))
+                            :content    (i18n/label :t/use-valid-qr-code {:data data})}})))
 
 (fx/defn handle-qr-code
```

Comment:
so looks like there is no need to have `handle-qr-code` at all, you can use  `process-qr-code` instead

Reply:
@krisc Can you comment on that?

Reply:
fixed

Reply:
Great. Makes sense just to make it just one function if one pretty much only calls the other.